### ISSUE TITLE
Create Custom BatchNormalization

### ISSUE THREAD

**Issue by MokkeMeguru** [MokkeMeguru/TFGENZOO#4]
Tensorflow's BatchNormalization Layer has no Reliability(complex implementation) and Scalability(for MultiGPU/TPU)

**Comment by MokkeMeguru** [comment]
Sync Batch was implemented in TF 2.2
